layman/overlays/source.py

```diff
@@ -111,6 +112,22 @@ def is_supported(self):
         '''Is the overlay type supported?'''
         return _supported(self.get_type_key(), self.supported)
 
+    def get_masters(self, base):
+        '''Return master repos, if present'''
+        lfile = path([base, self.parent.name, 'metadata', 'layout.conf'])
+        masters = []
+        if os.path.exists(lfile) and os.access(lfile, os.R_OK):
+            with fileopen(lfile, 'r') as f:
+                for l in f:
+                    s = l.split('=')
+                    if (len(s) < 2): continue
+                    k = s[0].strip()
+                    if (k == 'masters'):
+                        masters = '='.join(s[1:]).strip().split(' ')
```

Comment:
Generally, it's better to use split() instead of split(' '), since split() works correctly with multiple spaces and tabs.